neo4j-migrations-core/src/test/java/ac/simons/neo4j/migrations/core/PreconditionsIT.java

```diff
@@ -125,7 +127,7 @@ void thingsShouldNotFailWhenAssumptionsChangeDueToVersionUpgrade() {
 			assertStateBeforeAndAfterPreconditionChanged(driver);
 
 			try (Session session = driver.session()) {
-				session.writeTransaction(tx -> tx.run("CREATE(v:Version {name:'4.1'})").consume());
+				session.executeWrite(tx -> tx.run("CREATE(v:Version {name:'4.1'})").consume());
```

Comment:
```suggestion
				session.executeWriteWithoutResult(tx -> tx.run("CREATE(v:Version {name:'4.1'})"));
```

As an alternative option.